src/js/functions/sort/getTodosByPriority.js

```diff
@@ -0,0 +1,19 @@
+const getTodosByPriority = items => {
+  const duplicateItems = items.map(item => item);
+  const newItems = [];
+  const getPriorities = items.map(({ priority }) => priority);
+  const sortPriorities = getPriorities.sort((a, b) => a - b);
+  sortPriorities.forEach(sorted => {
+    let index = -1;
+    for (let i = 0; i < duplicateItems.length; i += 1) {
+      index += 1;
+      if (duplicateItems[i].priority === sorted) {
+        newItems.push(duplicateItems[i]);
+        duplicateItems.splice(index, 1);
+      }
+    }
+  });
+  return newItems;
+};
```

Comment:
We can have the two sort in one module instead of two different modules since they are doing almost the same thing, that is, sorting.

Reply:
This is the logic for sorting todos and is different from the one in your comment above as that one is for DOM manipulation